source/operations/checklists/hardware.rst

```diff
@@ -213,11 +213,17 @@ Storage
 
    Format drives as XFS and present them to MinIO as a :abbr:`JBOD (Just a Bunch of Disks)` array with no RAID or other pooling configurations.
 
-   MinIO **strongly recommends** disabling retry-on-error (``EIO``, ``ENOSPC``, and ``default``) behavior in XFS.
-   The default settings typically allow infinite retry-on-error and can produce unnecessary performance loss.
+   MinIO **strongly recommends** disabling `retry-on-error <https://docs.kernel.org/admin-guide/xfs.html?highlight=xfs#error-handling>`__ behavior using the ``max_retries`` configuration for the following error classes:
+   
+   - ``EIO`` Error when reading or writing
+   - ``ENOSPC`` Error no space left on device
+   - ``default`` All other errors
+
+   The default ``max_retries`` setting typically direct the filesystem to retry-on-error indefinitely instead of propagating the error.
```

Comment:
Setting is the singular noun here, so the verb needs to be singular, too.

```suggestion
   The default ``max_retries`` setting typically directs the filesystem to retry-on-error indefinitely instead of propagating the error.
```